requirements/base.txt

```diff
@@ -22,3 +22,4 @@ six==1.10.0
 pytest==3.6.0
 pytest-cov==2.5.1
 pytest-django==3.2.1
+supervisor==4.0.4
```

Comment:
Shouldn't this go to the production.txt?

Reply:
dropped supervisor